src/run.ts

```diff
@@ -0,0 +1,73 @@
+import { createClient } from "@supabase/supabase-js";
+import { CommanderError } from "commander";
+import { Octokit } from "@octokit/rest";
+import { createAdapters } from "./adapters";
+import { CommandParser } from "./handlers/command-parser";
+import { Context } from "./types/context";
+import { Database } from "./types/database";
+import { Env } from "./types/env";
+import { PluginInputs } from "./types/plugin-input";
+
+export async function run(inputs: PluginInputs, env: Env) {
+  if (inputs.eventName !== "issue_comment.created") {
+    console.warn(`Unsupported event ${inputs.eventName}, skipping.`);
```

Comment:
Should only be using our logger. Never console. How will this be debugged remotely?